proto/pocket/supplier/tx.proto

```diff
@@ -1,7 +1,18 @@
 syntax = "proto3";
+
 package pocket.supplier;
 
 option go_package = "pocket/x/supplier/types";
 
 // Msg defines the Msg service.
-service Msg {}
\ No newline at end of file
+service Msg {
+  rpc CreateClaim (MsgCreateClaim) returns (MsgCreateClaimResponse);
+}
+message MsgCreateClaim {
+  string supplierAddress = 1;
+  bytes smstRootHash = 2;
+  string sessionId = 3;
```

Comment:
Should we transition to using session_header?
cc @Olshansk

Reply:
Yes, as discussed, let's have a common `SessionHeader` protobuf message type defined in the session module which will be common to `RelayMets`, `MsgCreateClaim`, and `MsgSubmitProof`.